app/controllers/idv/in_person/address_controller.rb

```diff
@@ -29,7 +29,6 @@ def update
           attrs.each do |attr|
             pii_from_user[attr] = flow_params[attr]
           end
-          flow_session['Idv::Steps::InPerson::AddressStep'] = true
           redirect_to_next_page
         else
           render :show, locals: extra_view_variables
```

Comment:
(I'm still getting notification emails for this PR, so dropping in to comment.)

I would recommend updating next_steps after the delete PR is deployed to avoid 50/50 state issues, and maybe waiting until the state_id step is also out of the FSM. The whole issue of the FlowPolicy and back button for IPP needs a careful look before making changes.

Here's a [branch](https://github.com/18F/identity-idp/tree/ipp-back-button) that @gina-yamada, @JackRyan1989, and I worked on to try to improve that, and our conclusion was that there are several interacting bugs that need to be fixed in the state_id step and in_person_controller before the FlowPolicy & back button can be fully implemented. I would prioritize getting this merged, the [state_id_controller](https://github.com/18F/identity-idp/pull/9926) ready, and the Flow State Machine fully decommissioned before trying to integrate IPP and state_id and address into FlowPolicy. @solipet may also have a good perspective on this.

Reply:
Sonia I was just wishing I could ask you about this. Thanks so much for clarifying! I'll hold off on making that update then until we've fully removed fsm.

Reply:
I thought about IPP's interaction with FlowPolicy some more. After this delete PR is merged & deployed,
- I think it's worth fixing the bugs in state_id step (sequentially redirecting to multiple places in update) and in_person_controller (don't remember the details, but when you fix the state_id bug it makes some other test fail) as a first step.
- Since state_id is the only step left in in_person_controller, maaaaybe you could put a step_info method in in_person_controller and put ipp_state_id in document_capture_controller's next_steps. You'd have to try it and see how that works overall. Check the [state_id_controller branch](https://github.com/18F/identity-idp/pull/9926) for what that step_info might look like.
- Then you could use the confirm_step_allowed before_action in all the IPP controllers and remove the before_actions that prevent the back button.

You could talk with @solipet about getting FlowPolicy to work better with forks in the flow, and @jmhooper might have some thoughts too.